### DISCUSSION TITLE
Leaderboard for top contributors

### DISCUSSION THREAD

**Discussion by harinij** [outline/outline#1318] [General]
A feature that we would love to have is of leaderboards, that gives users motivation to contribute and engage with outline more. Maybe monthly top contributors report or overall etc. 

We were thinking of using the audit logs to see if we can do something with it, but thought will post here and check with the community if this already exists (and I somehow missed it) or there are alternative ways to be able to achieve this.

PS: Great product that we enjoy using :)

**Comment by tommoor** [comment]
Hey @harinij – good question, there isn't anything like this right now but it could be neat. It'd be interesting to dig in and figure out if there are other ways we could increase folks motivation to contribute too…

For example I find it quite motivating when I know other team members are reading the docs I wrote, perhaps a personal dash that shows activity on docs you've contributed to. Or a way for team members to leave props/kudos/likes on useful documents.
